<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TexinfoUi</class>
 <widget class="QDialog" name="TexinfoUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <widget class="QListWidget" name="fileListLW"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closePB">
       <property name="toolTip">
        <string>Close this dialog</string>
       </property>
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="rescanPB">
       <property name="toolTip">
        <string>Rebuild the file lists</string>
       </property>
       <property name="text">
        <string>&amp;Rescan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewPB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Show contents of marked file. Only possible when files are shown with path</string>
       </property>
       <property name="text">
        <string>&amp;View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="whatStyleCO">
       <property name="toolTip">
        <string>Selected classes or styles</string>
       </property>
       <item>
        <property name="text">
         <string>LaTeX classes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LaTeX styles</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BibTeX styles</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BibTeX databases</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Biblatex bibliography styles</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Biblatex citation styles</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="pathCB">
       <property name="toolTip">
        <string>Toggles view of the file list</string>
       </property>
       <property name="text">
        <string>Show &amp;path</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>whatStyleCO</tabstop>
  <tabstop>pathCB</tabstop>
  <tabstop>fileListLW</tabstop>
  <tabstop>rescanPB</tabstop>
  <tabstop>viewPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
